src/llnode.cc

```diff
@@ -97,30 +110,37 @@ bool BacktraceCmd::DoExecute(SBDebugger d, char** cmd,
   if (number != -1) num_frames = number;
   for (uint32_t i = 0; i < num_frames; i++) {
     SBFrame frame = thread.GetFrameAtIndex(i);
-    SBSymbol symbol = frame.GetSymbol();
-
-    // C++ symbol
-    if (symbol.IsValid()) {
-      SBStream desc;
-      if (!frame.GetDescription(desc)) continue;
-      result.Printf(frame == selected_frame ? "  * %s" : "    %s",
-                    desc.GetData());
-      continue;
+    const char star = (frame == selected_frame ? '*' : ' ');
+    const uint64_t pc = frame.GetPC();
+
+    if (!frame.GetSymbol().IsValid()) {
+      v8::Error err;
+      v8::JSFrame v8_frame(&llv8, static_cast<int64_t>(frame.GetFP()));
+      std::string res = v8_frame.Inspect(true, err);
+      if (err.Success()) {
+        result.Printf("  %c frame #%u: 0x%016llx %s\n", star, i, pc,
+                      res.c_str());
+        continue;
+      }
     }
 
-    // V8 frame
-    v8::Error err;
-    v8::JSFrame v8_frame(&llv8, static_cast<int64_t>(frame.GetFP()));
-    std::string res = v8_frame.Inspect(true, err);
-
-    // Skip invalid frames
-    if (err.Fail()) continue;
+#ifdef LLDB_SBMemoryRegionInfoList_h_
+    // Heuristic: a PC in WX memory is almost certainly a V8 builtin.
```

Comment:
Should we ask V8 team for some post-mortem information to be able to identify them?